templates/index.html

```diff
@@ -45,7 +45,8 @@
 
   $(document).ready(function() {
     // default iframe src with SymPy initcode
-    var iframe_src = "{{ host }}{{ path }}{{ query }}&code={{ initcode|urlencode }}";
+    // see https://jupyterlite.readthedocs.io/en/stable/quickstart/embed-repl.html#configuration for URL customisation
+    var iframe_src = "{{ host }}{{ path }}{{ query }}?kernel=python&execute=0?toolbar=1&code={{ initcode|urlencode }}";
```

Comment:
```suggestion
    var iframe_src = "{{ host }}{{ path }}{{ query|safe }}&code={{ initcode|urlencode }}";
```

Reply:
This is because `kernel=python` and `toolbar=1` are already in the `query` string defined on line 2 of the file.

If we wanted to add `execute=0`, it would make more sense to add it there.

I have added the Jinja filter `safe` so the query variable will be used [without any escaping](https://jinja.palletsprojects.com/en/stable/templates/#jinja-filters.safe) applied.

Reply:
Ah, I missed that – thanks for pointing it out and for this suggestion, addressed in 26a8cca8e914c4b19d3f4cdefe3cea0f43a57bac.